docs/workflows/sdk-setup.mdx

```diff
@@ -0,0 +1,292 @@
+---
+title: SDK Setup
+sidebar_position: 8
+# Display h2 to h2 headings
+toc_min_heading_level: 2
+toc_max_heading_level: 2
+---
+
+import styles from './css/styles.module.css';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
```

Comment:
nit: should we put a title up here too?

Reply:
I'm not sure I follow. There is a  title (`title: SDK Setup`) and a first header (`# Overview`). Are either of these the titles you are referring to?

Reply:
That title doesn't show on the screen though. So I was thinking a title on the page that you see once you click into it.

Reply:
Got it, I see what you did.